static/bidder-info/qt.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://endpoint1.qt.io/pserver"
+maintainer:
+  email: "qtssp-support@qt.io"
```

Comment:
Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
received response on email thread